packages/components/table/src/util.ts

```diff
@@ -251,10 +251,20 @@ export function toggleRowStatus<T>(
 
   const addRow = () => {
     statusArr.push(row)
+    if (row.children) {
```

Comment:
It does not seem to work on multi-layer tree

Reply:
now it works